fe/fe-core/src/main/java/com/starrocks/sql/optimizer/Optimizer.java

```diff
@@ -149,6 +149,8 @@ void memoOptimize(ConnectContext connectContext, Memo memo, TaskContext rootTask
             context.getRuleSet().addMergeJoinImplementationRule();
         } else if ("hash".equalsIgnoreCase(joinImplementationMode)) {
             context.getRuleSet().addHashJoinImplementationRule();
+        } else if ("cross".equalsIgnoreCase(joinImplementationMode) || "nestloop".equalsIgnoreCase(joinImplementationMode)) {
```

Comment:
not use nest loop join by default?

Reply:
cross looks not a join implemtation, it's a join type

Reply:
+1